### PR TITLE
Add collection rancheros information script

### PR DESCRIPTION
**Pull request by kingsd041**
This script is used to collect rancheros logs and configuration.

### CODE REVIEW

File: scripts/tools/collect_rancheros_info.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# How to use
+#
+# 1. Login to your rancheros and switch to root
+#    $ sudo su - root
+# 2. Collecting rancheros information
+#    # curl https://raw.githubusercontent.com/rancher/os/master/scripts/tools/collect_rancheros_info.sh | sh
+
+set -e
+# /var/log directory
+boot_log_src_dir=/var/log
```
Comment (niusmallnan) [inline scripts/tools/collect_rancheros_info.sh]: log_src_dir

File: scripts/tools/collect_rancheros_info.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+for i in $dest_conf_dir $dest_log_dir;
+do
+  if [ ! -d $i ]; then
+    mkdir -p  $i
+  fi
+done
+
+# Hidden ssh-rsa
+hiddenSshRsa(){
+     sed -i 's/ssh-rsa.*$/ssh-rsa .../g' $1
```
Comment (niusmallnan) [inline scripts/tools/collect_rancheros_info.sh]: indent

File: scripts/tools/collect_rancheros_info.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+hiddenSshRsa $dest_conf_dir/ros-config-export.conf
+if [ -f  $dest_conf_dir/metadata ]; then
+    hiddenSshRsa $dest_conf_dir/metadata
+fi
+
+tar -c -f /tmp/rancheros_export_$ARCHIVE -C $dest_dir  . >/dev/null 2>&1
+
+/bin/echo "*********************************************************"
+/bin/echo -e "The RancherOS config and log are successfully exported. \
+\nPlease check the /tmp/rancheros_export_$ARCHIVE."
```
Comment (niusmallnan) [inline scripts/tools/collect_rancheros_info.sh]: just use `echo`

File: scripts/tools/collect_rancheros_info.sh (hunk 1 above)
Comment (kingsd041) [reply scripts/tools/collect_rancheros_info.sh]: updated

File: scripts/tools/collect_rancheros_info.sh (hunk 2 above)
Comment (kingsd041) [reply scripts/tools/collect_rancheros_info.sh]: updated

File: scripts/tools/collect_rancheros_info.sh (hunk 3 above)
Comment (kingsd041) [reply scripts/tools/collect_rancheros_info.sh]: updated

**Pull request merged**